versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,158 @@
+  cattle-csp-billing-adapter-system   rancher-csp-billing-adapter   1/1     1            1           30m
+  cattle-fleet-local-system           fleet-agent                   1/1     1            1           29m
+  cattle-fleet-system                 fleet-controller              1/1     1            1           29m
+  cattle-fleet-system                 gitjob                        1/1     1            1           29m
+  cattle-provisioning-capi-system     capi-controller-manager       1/1     1            1           28m
+  cattle-system                       rancher                       1/1     1            1           32m
+  cattle-system                       rancher-webhook               1/1     1            1           29m
+  cert-manager                        cert-manager                  1/1     1            1           32m
+  cert-manager                        cert-manager-cainjector       1/1     1            1           32m
+  cert-manager                        cert-manager-webhook          1/1     1            1           32m
+  ingress-nginx                       ingress-nginx-controller      1/1     1            1           33m
+  kube-system                         coredns                       2/2     2            2           38m
+  ```
+
+### Check Helm Chart Installation
+
+Check that the Helm chart installation is complete:
+
+```shell
+helm ls -n cattle-rancher-csp-deployer-system
+```
+
+After completing the Helm chart installation, you can verify the installation was successful:
+
+```shell
+helm status rancher-cloud -n cattle-rancher-csp-deployer-system
+```
+
+Refer to the [Troubleshooting](troubleshooting.md) section for a failed installation.
+
+After the Helm chart installation is complete, Rancher Prime is successfully installed.
+
+## Log into the Rancher Dashboard
+
+You may now log in to the Rancher dashboard by pointing your browser to the Rancher server URL `https://$RANCHER_HOSTNAME`, where`$RANCHER_HOSTNAME` is the hostname chosen when [installing Rancher](#installing-rancher).
+
+:::note
+
+The Rancher hostname must be resolvable by public DNS. For more details, please refer to the [Prerequisites](prerequisites.md) section.
+
+:::
+
+## Uninstalling Rancher Prime PAYG Offering
+
+Run the following command to uninstall Rancher Prime:
+
+```shell
+helm uninstall -n cattle-rancher-csp-deployer-system rancher-cloud
+```
+
+Uninstalling Rancher Prime may not cleanly remove all Kubernetes resources that Rancher created. You can use the [Rancher resource cleanup script](https://github.com/rancher/rancher-cleanup) to perform a more comprehensive cleanup.
+
+The best practice for uninstalling the Rancher Prime PAYG offering is to migrate any non-Rancher workloads to a different cluster and destroy the Rancher cluster.
+
+:::important
+Ensure you prepare and migrate any non-Rancher workloads off the cluster before destroying the cluster since these resources are not recoverable.
+:::
```

Comment:
`important` is not a built-in admonition so you need to use a custom label, e.g. `:::note important`, but since the content itself is about an unrecoverable action `danger` or `warning` may be more appropriate.